packages/lit-ssr/src/lib/util/parse5-utils.ts

```diff
@@ -0,0 +1,259 @@
+/**
+ * TODO(usergenic): The following set of helper functions are more-or-less
+ * copied from the npm package dom5 which could not be brought in at this
+ * time because it is bound to `parse5@4` where this package uses `parse5@5`.
+ * Once dom5 is updated, we can just use that package and not maintain these
```

Comment:
Check if this has happened?

Reply:
Nope https://github.com/Polymer/tools/blob/master/packages/dom5/package.json#L22

Reply:
ok